process/slack_etiquette.md

```diff
@@ -0,0 +1,64 @@
+# **`Draft language. Needs Editing.`**
+
+This was copied (_with permission_) from the Codebuddies (www.codebuddies.org) community.  
+While some of these are **great** for PyLadies, some of these points need editing for our commuity.
+
+Please propose or edit as you see fit.
+
+______________________________________
+______________________________________
+
+## The following guidelines can help you get the most from our Slack community.
+
+.
+### :star2: Every member of CodeBuddies is a _*person*_ 
+ _This is a safe space. No question is "stupid.", no topic too "basic"._  
+.
+This community is intended for self-directed learners, CS students, programming enthusiasts, and professional developers to share knowledge and techniques. The admins, moderators, and everyone else in CodeBuddies are **donating their spare time; nobody’s being paid.**  Our suggestions/answers may not always be correct, & you may not always get an answer. Treat us like somebody friendly you don’t know who’s standing next to you offering help or suggestions.
+
+### :star2: CodeBuddies is asynchronous & international
+We have members on every continent (except Antartica), and many of us are working professionals. You may get help within 5 minutes if somebody who knows the answer is online looking for something to do -- but it’s entirely possible that you won’t get a response for an hour or day or more. If you’re asking about a less popular tool, or a specific edge case, you may even get an answer _**several days later.**_
+.
+So remember to keep CodeBuddies open in the background for a while after asking a question - or check back periodically.
+.
+Please don't cross post the same question in multiple channels. People in this community may not be super-chatty at any given moment, but we are reading posts across all channels.  Someone will eventually see (and help) you. _**We promise**_.
+
+### :star2: Some channels are more active than others
+Questions in #general or #advice tend to get answered quickly, or they get pushed off the screen & forgotten. We have a handful of dedicated people that read the scrollback and try to answer forgotten questions, but there’s a lot of discussion/movement happening, you may want to re-post after a few days.  
```

Comment:
*Edit:*

`Questions in #general tend to get answered quickly, or they get pushed off the screen & forgotten. While sometimes members will revisit a conversation if it has gone offscreen it can be that when there's a lot of discussion/movement happening, you may want to re-post after a few days.`